doc/reference/api/overview.rst

```diff
@@ -256,6 +256,11 @@ current :ref:`stability level <api_lifecycle>`.
      - Experimental
      - 2.4
      - 2.4
+     
+   * - :ref:`reset_api`
+     - Experimental
+     - 3.0
+     - 3.0
```

Comment:
should be 3.1, since it will be the first release with this API

Reply:
Done!